### ISSUE TITLE
Accordion documentation should mention the use of hidden="true" 

### ISSUE THREAD

**Issue by gaurav5430** [w3c/aria-practices#2443]
On this page: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/

in the accessibility considerations, it is not mentioned that we should be using hidden="true" on all the collapsed panels, while the implementation / code actually uses this.

Would it make sense to add this info about using hidden="true" explicitly on the page?

**Comment by sivakusayan** [comment]
Not sure what the owners of this repository think, but I think this makes sense!
I've noticed that while I was at Epic Systems, many engineers (even tenured ones) assumed that it was a screen reader bug that an element with `height: 0px` CSS was still being read out.

Calling out that we use the `hidden` attribute to hide something in the AX tree would be helpful.
